LICENSE-binary

```diff
@@ -445,6 +445,8 @@ jline:jline
 org.jodd:jodd-core
 pl.edu.icm:JLargeArrays
 
```

Comment:
```suggestion
```

Reply:
Elsewhere in this file we have a empty line between Java and Python packages so this follows the existing style